metrics/metrics.go

```diff
@@ -0,0 +1,150 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/jtblin/kube2iam/version"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const namespace = "kube2iam"
+
+var (
+	// IamRequestCount counts number of outbound requests made to AWS IAM service.
```

Comment:
These comments above each metric are a bit redundant as the `Help` field describes the metric.

Reply:
This was to silence `golint`, I think I'll keep them just so that every file in the project gets no golint errors.

Reply:
👍